kafka/producer/base.py

```diff
@@ -27,6 +27,9 @@
 
 STOP_ASYNC_PRODUCER = -1
 
+# unlimited
+ASYNC_QUEUE_MAXSIZE = 0
```

Comment:
Large limit != unlimited. My point is that:

1. Callers should pass in the limit. If caller wants unbounded queue, it has to be explicitly set to 0.
2. And the library should be defensive, and impose a limit (though it's a large one).